OutOfSchool/OutOfSchool.DataAccess/Models/Section.cs

```diff
@@ -0,0 +1,59 @@
+﻿#nullable enable
```

Comment:
why do we need this line?

Reply:
![image](https://user-images.githubusercontent.com/55458556/107985273-74e65700-6fd2-11eb-9edb-8883b9e00c3f.png)
I specified that Facebook, Instagram and Website properties can be null using "?" sign. Then Rider suggested me to add this line to the file. 
I always do it in this way. Is it not necessary?

Reply:
it's ok, thanks!